spec/unit/defines/server/role_spec.rb

```diff
@@ -59,12 +59,13 @@
     end
 
     it { is_expected.to contain_postgresql__server__role('test') }
-    it 'has create role for "test" user with password as ****' do
+    it 'has create role for "test" user with password as ****' do # rubocop:disable RSpec/ExampleLength
```

Comment:
The [regression test suite failure](https://travis-ci.org/puppetlabs/puppetlabs-postgresql/jobs/350248274) seems related to this change